op/commander.go

```diff
@@ -37,7 +40,8 @@ func (c Commander) Run() (interface{}, Status, error) {
 	// Execute command
 	bts, err := exec.Command(cmd, args...).CombinedOutput()
 	if err != nil {
-		return string(bts), Unknown, CommandExecError{command: c.Command, format: c.format, err: err}
+		err1 := CommandExecError{command: c.command, format: c.format, err: err}
+		return New(c, string(bts), Unknown, err1)
```

Comment:
I know it didn't change in this PR, but should we change `New` to `NewOp`? There are several different structs in package `op`, and I wonder if it would be easier to read if the name said what we're creating. (I say that as someone who had to look up what `New` was when reviewing this PR. 😄)